.github/workflows/greeting.yml

```diff
@@ -0,0 +1,33 @@
+name: Greetings for All
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened,closed]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create comment for issue
+      if: github.event_name =='issues'
+      uses: peter-evans/create-or-update-comment@v1
+      with:
+        issue-number: ${{tojson(github.event.issue.number)}}
+        body: |
+            @${{ github.actor }} Thanks for opening the New Issue, This Will definitely Improve our Project.
```

Comment:
Please append this line as well:
```Use /assign to self assign this issue```

Preferably in bolds

Reply:
Ook